### PR TITLE
Delete 2019 code

### PR DESCRIPTION
**Pull request by antoinealb**
This PR removes the code that was specific to the 2019 robot so that we can all start working on the 2020 project :)

@SyrianSpock: it removes a lot of your code, not too sad to see it go ?

### CODE REVIEW

File: master-firmware/src/strategy.cpp [hunk 1]
```diff
@@ -168,53 +105,23 @@ static void wait_for_autoposition_signal(void)
...
-    return res;
-}
-
-void strategy_shutdown_endgame(void)
-{
-    manipulator_gripper_set(BOTH, GRIPPER_OFF);
-}
-
-void strategy_order_play_game(strategy_context_t* ctx, RobotState& state)
+void strategy_order_play_game(RobotState& state, enum strat_color_t color)
```
Comment (SyrianSpock) [inline master-firmware/src/strategy.cpp]: pointers are more in your face IMHO

Comment (antoinealb) [reply master-firmware/src/strategy.cpp]: Not sure I understand the comment here

Comment (SyrianSpock) [reply master-firmware/src/strategy.cpp]: I would pass `RobotState` as a pointer instead of a reference

Comment (SyrianSpock) [reply master-firmware/src/strategy.cpp]: But then it could be null #NoFreeLunch 😭

Comment (antoinealb) [reply master-firmware/src/strategy.cpp]: Yes I think I will keep as non nullable for now and merge it like this.

Comment (SyrianSpock) [reply master-firmware/src/strategy.cpp]: :shipit:

**Pull request merged**